its_on/models.py

```diff
@@ -17,6 +18,13 @@
     sa.Column('groups', postgresql.ARRAY(sa.String(255))),
     sa.Column('version', sa.Integer, nullable=True),
     sa.Column('comment', sa.Text),
+    sa.Column('created_at', sa.DateTime, default=lambda: datetime.datetime.now(), nullable=True),
+    sa.Column(
+        'updated_at', sa.DateTime,
+        default=lambda: datetime.datetime.now(),
+        onupdate=lambda: datetime.datetime.now(),
```

Comment:
Same here about using DB server datetime instead of application server's local one